modules/eco-node-health-check-operator-about.adoc

```diff
@@ -58,7 +58,36 @@ During the upgrade process, nodes in the cluster might become temporarily unavai
 [id="understanding-nhc-operator-workflow_{context}"]
 == Understanding the Node Health Check Operator workflow
 
-When a node is identified as unhealthy, the Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the node's health status is updated accordingly.
+When a node is identified as unhealthy, the Node Health Check Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the node's health status is updated accordingly.
 
 When the node turns healthy, the controller deletes the external remediation template
-and updates the node's health status.
\ No newline at end of file
+and updates the node's health status.
+
+[id="preventing-nhc-conflict-with-mhc_{context}"]
+== Preventing node health check conflicts with machine health check
+
+When both, node health check and machine health check are deployed, the node health check avoids conflict with the machine health check.
+
+The following list summarizes the system behavior when node health checks and machine health checks are deployed:
+
+* If only the default machine health check exists, the node health check continues to identify unhealthy nodes. However, the node health check ignores unhealthy nodes in a `Terminating` condition. The default machine health check handles the unhealthy nodes with a `Terminating` condition.
```

Comment:
I know we used "default" all the time... but I'm wondering if it's clear for others what we mean with it. Maybe we can add the the name of the default MHC: "machine-api-termination-handler"?

Reply:
@slintes Makes sense. To address this, I have added the following note:
_OpenShift Container Platform deploys `machine-api-termination-handler` as the default `MachineHealthCheck` resource._